builds/checkin/edgelet.yaml

```diff
@@ -45,7 +45,7 @@ jobs:
         displayName: Set Version
       - script: edgelet/build/linux/install.sh
         displayName: Install Rust
-      - script: 'cargo install --git https://github.com/myagley/cross.git --branch set-path'
+      - script: 'cargo install --git https://github.com/arsing/cross.git --branch set-path'
```

Comment:
This GitHub URL change was intentional?

Reply:
Yes, it's [this commit](https://github.com/Azure/iotedge/pull/896/commits/08394d986954a5162b8492443e5e37c24a587606) from the original PR. Looks like I forgot to add it to the commit message back then.